code/modules/events/event_container.dm

```diff
@@ -21,11 +21,17 @@ GLOBAL_LIST_EMPTY(event_last_fired)
 	var/datum/event_meta/next_event = null
 
 	var/last_world_time = 0
+	var/minimum_players = 15 // Minimum amount of crew for events to kick off
 
 /datum/event_container/process()
 	if(!next_event_time)
 		set_event_delay()
 
+	if(SSticker.mode.num_players_started() < minimum_players)
+		delayed = TRUE
+	else
+		delayed = FALSE
```

Comment:
I think `/datum/event_container/process()` is called often when the Events subsystem fires:
https://github.com/ScorpioStation/ScorpioStation/blob/master/code/controllers/subsystem/events.dm#L41

The effect of this code would be that:
- Events cannot be enabled when 0-14 players are returned by `num_players_started()`
- Events cannot be disabled when 15+ players are returned by `num_players_started()`

I think Admins should retain at least as much control as they have now over event management.

Being unable to manually pause/resume the event levels makes it more difficult to:
- Schedule desired events on low-pop; the admin has to manually fire the desired event at the time they want it to happen
- Suppress undesired events at higher-pop; the admin has to increase the timer and/or explicitly choose Nothing as the desired event

Perhaps change this to set the roundstart default (delayed = FALSE / TRUE) based on roundstart population instead?

Reply:
Done!